src/components/Auth/RequiresAuth.jsx

```diff
@@ -0,0 +1,7 @@
+import { useAuth } from "../../context/auth-context";
```

Comment:
instead of giving relative path we can give absolute path,i have used in one app it gives more readability and easy to write